### PR TITLE
ruRU update.

### PR DESCRIPTION
**Pull request by Wetxius**

### CODE REVIEW

File: Plugins/Locales/ruRU.lua [hunk 1]
```diff
@@ -11,30 +11,30 @@ L.positionDesc = "Введите в поле или передвиньте як
...
+L.test = "Тест"
+L.altPowerTestDesc = "Показать монитор 'Дополнительной Энергии', позволяя перемещать его, и симулировать изменение энергии как на сражении с боссом."
+L.yourPowerBar = "Ваша полоса Энергии"
+L.barColor = "Цвет полосы"
+L.barTextColor = "Цвет текста полосы"
+L.additionalWidth = "Дополнительная ширина"
+L.additionalHeight = "Дополнительная высота"
+L.additionalSizeDesc = "Увеличьте размер стандартного монитора с помощью ползунка или введите значение вручную в поле, если оно выше 100."
+L.yourPowerTest = "Ваша энергия: %d" -- Your Power: 42
+L.yourAltPower = "%s: %d" -- e.g. Your Corruption: 42
```
Comment (funkydude) [inline Plugins/Locales/ruRU.lua:35]: Why remove "your" from this @Wetxius ?

Comment (Wetxius) [reply Plugins/Locales/ruRU.lua:35]: For different abilities, it is necessary to do a different translation depending on the gender of the word: feminine / masculine / neuter. 
"Ваша порча" -Your Corruption
But  "Ваше отравление" - Your Poison

Comment (Wetxius) [reply Plugins/Locales/ruRU.lua:35]: It is better to leave only the name of the ability so that there is no bad word combination.

Comment (funkydude) [reply Plugins/Locales/ruRU.lua:35]: @Wetxius So you're saying in Russian the words "Corruption" and "Poison" have a gender? And that "Ваше" or "Ваша" doesn't relate to your own personal gender?

Comment (Wetxius) [reply Plugins/Locales/ruRU.lua:35]: Yes. If you want to relate to own, you should write "Your level Corruption/Poison". It translate independent of gender like "Ваш уровень Порчи/Яда".

Comment (funkydude) [reply Plugins/Locales/ruRU.lua:35]: Thanks for explaining :)

**Pull request merged**